src/core_plugins/kibana/common/tutorials/logstash_instructions.js

```diff
@@ -15,12 +17,15 @@ export const LOGSTASH_INSTRUCTIONS = {
     ],
     WINDOWS: [
       {
-        title: 'Download and install the Java runtime environment',
+        title: 'Download and install the Java Runtime Environment',
         textPre: 'Follow the installation instructions [here](https://docs.oracle.com/javase/8/docs/technotes/guides/install/windows_jre_install.html).'
       },
       {
         title: 'Download and install Logstash',
-        textPre: 'Download Logstash from [here](https://artifacts.elastic.co/downloads/logstash/logstash-{config.kibana.version}.zip) and unzip it.'
+        textPre: 'First time using Logstash?  See the ' +
+          '[Getting Started Guide]({config.docs.base_url}guide/en/logstash/current/getting-started-with-logstash.html).\n' +
+          '  1. Download the Logstash Windows zip file from the [Downloads page]({config.docs.base_url}downloads/logstash).\n' +
```

Comment:
Why are we sending the user to the downloads page instead of directly to the zip file link? This means the user has to make another click to download the zip file.

Reply:
Chatted with @gchaps on Slack about this. Summarizing our conversation here:

Another issue with sending the users to the download page (as opposed to the direct versioned link to the artifact) is that the download page always shows the latest version. So, for example, if Kibana 6.3 and 7.1 are released and the user is viewing this tutorial in 6.3, they might accidentally end up downloading Logstash 7.1, which might not work with their 6.3 setup.  There is a link to past releases on the download page, but that's one more click + the user would have to realize that they are on the wrong version to even think of looking at past releases.

The downside of linking directly to the versioned artifact is that we might decide to change the artifact link and forget to update the tutorial content to match.

I don't know if there's a perfect solution here but, between the two options, I think the direct versioned link to the artifact is better.